plugins/woocommerce-blocks/tests/e2e/utils/wp-cli.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { promisify } from 'util';
+import { exec } from 'child_process';
+
+const execPromisified = promisify( exec );
+
+/**
+ * Runs a WP-CLI command inside the tests-cli container.
+ */
+export async function wpCLI( command: string ) {
+	return await execPromisified(
```

Comment:
NIT: It is not necessary `await` the return for async function
```suggestion
	return execPromisified(
```